packages/wrangler/src/cfetch/internal.ts

```diff
@@ -33,23 +34,39 @@ export async function fetchInternal<ResponseType>(
   addAuthorizationHeader(headers, apiToken);
 
   const queryString = queryParams ? `?${queryParams.toString()}` : "";
+  const method = init.method ?? "GET";
```

Comment:
So this change makes no behavioural difference, right?
Was it just a personal aesthetic thing?

Reply:
It's used in the error message here: https://github.com/cloudflare/wrangler2/pull/419/files#diff-c3f5c8f3aadedf1f97770eca2de93b3c5be926475a0f2618e59ad1955a840892R57

Reply:
Ahah!